src/Cms/AppErrors.php

```diff
@@ -158,9 +158,27 @@ protected function setWhoopsHandler($handler): void
         $whoops = $this->whoops();
         $whoops->clearHandlers();
         $whoops->pushHandler($handler);
+        $whoops->pushHandler($this->getExceptionHookWhoopsHandler());
         $whoops->register(); // will only do something if not already registered
     }
 
+    /**
+     * Initialize a callback handler for triggering the system.exception hook
+     *
+     * @return CallbackHandler
+     */
+    protected function getExceptionHookWhoopsHandler(): CallbackHandler
+    {
+        return new CallbackHandler(function ($exception, $inspector, $run) {
+            $this->trigger('system.exception', [
+                'exception' => $exception,
+                'kirby' => $this
+            ]);
```

Comment:
The `$kirby` argument is not needed as the Kirby object is bound to `$this` inside the hook callback.

```suggestion
            $this->trigger('system.exception', compact('exception'));
```